integration-libs/cpq-quote/root/components/cpq-quote-heading/cpq-quote-heading.component.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, Input, OnDestroy, OnInit, Optional } from '@angular/core';
+import { OrderEntry } from '@spartacus/cart/base/root';
+import { OutletContextData } from '@spartacus/storefront';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'cx-cpq-quote-heading',
+  templateUrl: './cpq-quote-heading.component.html',
+})
+export class CpqQuoteHeadingComponent implements OnInit, OnDestroy {
+  @Input()
+  quoteDiscountData: OrderEntry;
+  protected subscription = new Subscription();
+
+  constructor(@Optional() protected outlet: OutletContextData) {}
+  discountLabel: string = 'Discount Percentage';
```

Comment:
MAJOR: can we add this label into translations file and use it instead of hard coding it here?